pkg/transform/clusterquota/clusterquota.go

```diff
@@ -0,0 +1,25 @@
+package clusterquota
+
+import (
+	o7tapiquota "github.com/openshift/api/quota/v1"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// BuildManifest definitions
+func BuildManifest(quota o7tapiquota.ClusterResourceQuota) (*o7tapiquota.ClusterResourceQuota, error) {
```

Comment:
Similar functions for oauth and project are called `Translate`, I prefer us to keep similar names in such cases. You can rename this to `Translate` or rename other ones to` BuildManifest`

Reply:
Consider it done!